src/hearings/store/reducers/hearing-request.reducer.ts

```diff
@@ -23,6 +24,8 @@ export const initialHearingRequestState: HearingRequestStateData = {
       listingComments: null,
       hearingRequester: null,
       leadJudgeContractType: null,
+      cancelationReason: [],
+      cancelledCaseHearing: null,
```

Comment:
this hearing request reducer is used for request hearing only, the cancel shouldn't use ngrx store.